### ISSUE TITLE
[cmake] tarball-env-shell emits gsl error (Trac #2226)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2218]
I get the following error whenever I use a tarball

```text
/data/user/kmeagher/simulation.trunk.r166821.Linux-x86_64.gcc-7.3.0/env-shell.sh: line 84: gsl-config_: command not found
```
This is caused by bad code in `tarball-env.sh.in`:

```text
82	_GSL_FOUND=@GSL_FOUND@
83	if [[ "${_GSL_FOUND}" == "TRUE" ]]; then
84	    GSL_LIBRARY_DIR=`gsl-config_@GSL_VERSION@ --prefix`/lib/gsl-@GSL_VERSION@
85	    if [[ -d ${GSL_LIBRARY_DIR} ]]; then
86	        _LD_LIBRARY_PATH=${_LD_LIBRARY_PATH}:${GSL_LIBRARY_DIR}
87	        _DYLD_LIBRARY_PATH=${_DYLD_LIBRARY_PATH}:${GSL_LIBRARY_DIR}
88	    fi
89	fi
```
apparently `@GSL_VERSION@` is not set which causes the shell to run `gsl-config_` with a trailing underscore. This doesn't matter because gsl is a directory which is in LD_LIBRARTY_PATH. I think this is all detritus from the old ports system and can be removed.

Migrated from https://code.icecube.wisc.edu/ticket/2226
```json
{
    "status": "closed", 
    "changetime": "2019-03-12T17:04:47", 
    "description": "I get the following error whenever I use a tarball\n{{{\n/data/user/kmeagher/simulation.trunk.r166821.Linux-x86_64.gcc-7.3.0/env-shell.sh: line 84: gsl-config_: command not found\n}}}\nThis is caused by bad code in `tarball-env.sh.in`:\n\n{{{\n82\t_GSL_FOUND=@GSL_FOUND@\n83\tif [[ \"${_GSL_FOUND}\" == \"TRUE\" ]]; then\n84\t    GSL_LIBRARY_DIR=`gsl-config_@GSL_VERSION@ --prefix`/lib/gsl-@GSL_VERSION@\n85\t    if [[ -d ${GSL_LIBRARY_DIR} ]]; then\n86\t        _LD_LIBRARY_PATH=${_LD_LIBRARY_PATH}:${GSL_LIBRARY_DIR}\n87\t        _DYLD_LIBRARY_PATH=${_DYLD_LIBRARY_PATH}:${GSL_LIBRARY_DIR}\n88\t    fi\n89\tfi\n}}}\napparently `@GSL_VERSION@` is not set which causes the shell to run `gsl-config_` with a trailing underscore. This doesn't matter because gsl is a directory which is in LD_LIBRARTY_PATH. I think this is all detritus from the old ports system and can be removed.", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1552410287216031", 
    "component": "cmake", 
    "summary": "[cmake] tarball-env-shell emits gsl error", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2018-12-17T17:41:56", 
    "milestone": "Vernal Equinox 2019", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:59:24: 
* olivas changed owner from "" to "nega"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20190312T17:04:47: 
* nega commented:

> In changeset 171933:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171933"
> fix gsl detection for tarballs
> 
> closes: #2218
> ```


* nega changed resolution from "" to "fixed"
* nega changed status from "assigned" to "closed"
